wiki/Storyboards/pt.md

```diff
@@ -1,12 +1,10 @@
 # Storyboards
 
-![Essa cena é mostrada em quanto joga Walkie Talkie Man. Porem é apenas mostrada quando se consegue um hit "Elite Beat 300"](img/Sbpassing.png "Essa cena é mostrada em quanto joga Walkie Talkie Man. Porem é apenas mostrada quando se consegue um hit \"Elite Beat 300\"")
-![Isso é mostrado quando você erra ou faz um hit 50 no combo mostrado anteriormente.](img/Sbfailing.png "Isso é mostrado quando você erra ou faz um hit 50 no combo mostrado anteriormente.")
+![Essa cena é mostrada em quanto joga Walkie Talkie Man. Porem é apenas mostrada quando se consegue um hit "Elite Beat 300"](img/Sbpassing.png "Essa cena é mostrada em quanto joga Walkie Talkie Man")
```

Comment:
```suggestion
![Essa cena é mostrada enquanto joga Walkie Talkie Man.](img/Sbpassing.png "Essa cena é mostrada enquanto joga Walkie Talkie Man")
```
Same as in pt-br
This extra part is not currently in the en.md in this PR
typos, from what I could tell it is correct/exists in both Portuguese usages but not written like this in this context

Reply:
they were supposed to be there, i just couldn't get them to work as i wanted last time, but now i figured that the quotation marks were breaking the images so i'll just not use them anymore.

i'll fix both "enquanto" though